backend/src/main/java/gov/cdc/usds/simplereport/service/DataHubUploaderService.java

```diff
@@ -95,17 +182,21 @@ private void _uploadCSVDocument(final String apiKey) throws IOException, RestCli
             return execution.execute(request, body);
         })).build();
 
-        _uploadResultId = restTemplate.postForObject(DATAHUB_UPLOAD_URL, contentsAsResource, String.class);
+        // have data hub only reject some lines not whole upload
+        URI url = UriComponentsBuilder.fromUriString(_config.getUploadUrl())
+                .queryParam("option", "SkipInvalidItems").build().toUri();
+
+        _resultJson = restTemplate.postForObject(url, contentsAsResource, String.class);
```

Comment:
To expand a very little (I was rushed then; I should now be asleep), we should catch exceptions and mark the data_hub_upload object FAILED rather than just leaving it hanging. And we could probably just get a Map back from RestTemplate, I don't recall exactly how that works.